ui/components/TracePage/Toolbar.tsx

```diff
@@ -0,0 +1,112 @@
+import { chain } from "lodash";
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  HStack,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Switch,
+} from "@chakra-ui/react";
+import { CallInfo, Calls, useTreeContext } from "/components/TracePage/TracePage";
+import { ArrowsIn, ArrowsOut, CaretDown } from "phosphor-react";
+import { CallFunction, CallName } from "/components/TracePage/CallName";
+
+const SelectHighlightedFunction = () => {
+  const { calls, highlighted, setHighlighted } = useTreeContext();
+  const nameCounts = chain(calls)
+    .values()
+    .slice(1) // skip the root
+    .filter("name")
```

Comment:
Yes, because in addition to the root (an actual call that is invisible in the trace) there's also a broken non-call in the `Calls` that has the ID of the trace as a side effect of the children thing. Alternatively I could `.slice(2)` which looks more confusing. Hopefully this'll be solved better later.

Reply:
Can you add a comment about the broken non-call thing?